hindi/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -15,4 +15,9 @@
 4.  उठो, जागो और तब तक रुको नही जब तक मंजिल प्राप्त न हो जाये ।
 
 
-[और देखिये](http://www.hindisahityadarpan.in/2011/11/great-quotations-by-swami-vivekananda.html)
\ No newline at end of file
+5. लोग तुम्हारी स्तुति करें या निन्दा, लक्ष्मी तुम्हारे ऊपर कृपालु हो या न हो, तुम्हारा देहान्त आज हो या एक युग मे, तुम न्यायपथ से कभी भ्रष्ट न हो।

```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.